### PR TITLE
Added the Link to code of conduct in Readme

### PR DESCRIPTION
**Pull request by ramanjitsingh-hub**
The link was added to the readme file which now points to [this](https://github.com/organicmaps/organicmaps/blob/master/docs/CODE_OF_CONDUCT.md)
Signed-off-by: Ramanjit Singh

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -146,4 +146,4 @@ and [Google Play](https://play.google.com/store/apps/details?id=app.organicmaps)
 [Instagram](https://instagram.com/organicmaps.app/).
   - Güncellemelerimizi [Instagram](https://instagram.com/organicmapstr/) üzerinden takip edin.
 
-The Organic Maps community abides by the CNCF code of conduct.
+The Organic Maps community abides by the CNCF [code of conduct.](https://github.com/organicmaps/organicmaps/blob/master/docs/CODE_OF_CONDUCT.md)
```
Comment (biodranik) [inline README.md:149]:
```suggestion
The Organic Maps community abides by the CNCF [code of conduct](https://github.com/organicmaps/organicmaps/blob/master/docs/CODE_OF_CONDUCT.md).
```

**Review by biodranik — APPROVED** [review]
Thank you!

**Comment by biodranik** [comment]
Can you please squash commits and sign them with the DCO, like it says in the failed DCO check here?

**Comment by ramanjitsingh-hub** [comment]
I seriously don't know what is happening, I am using github desktop to do the squash

**Comment by ramanjitsingh-hub** [comment]
I am going to close this pull request and do it again

**Comment by biodranik** [comment]
@ramanjitsingh-hub Please DO NOT CLOSE pull requests. Learn how to squash your commits with the interactive rebase from the command line and how to force-push your changes back to the original PR.

**Pull request closed without merging**